src/services/telegram.ts

```diff
@@ -1,26 +1,16 @@
 import { Service } from "../common";
-import { SdkError } from "../types";
 
 export class TelegramService extends Service {
-  private get chatId(): string {
-    const chatId = process.env.TELEGRAM_CHAT_ID;
-    if (!chatId) {
-      throw new SdkError("Telegram Chat Id not set");
-    }
-    return chatId;
-  }
-
-  private get botId(): string {
-    const botId = process.env.TELEGRAM_BOT_ID;
-    if (!botId) {
-      throw new SdkError("Telegram Bot Id not set");
-    }
-    return botId;
-  }
-
   sendMessage(text: string) {
-    const params = new URLSearchParams({ chat_id: this.chatId, text: text, disable_web_page_preview: "true" });
-    const url = `https://api.telegram.org/bot${this.botId}/sendMessage?` + params;
+    if (!this.ctx.simulation.telegramChatId || !this.ctx.simulation.telegramBotId) {
+      return;
```

Comment:
@nymmrx do you think we should throw an error here? Just don't want it to be annoying if people choose not to use telegram

Reply:
I think no logs is better. We can add this to the docs